### PR TITLE
JW-491-502 MSI installer.

### PR DESCRIPTION
**Pull request by m-kostrzewa**
XML files copied over from example in go-msi repo. Only real changes are inside wix.json and LICENSE.rtf

### CODE REVIEW

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
To test:

- spawn a testbed
- Get-Service Contrail* should return nothing
- .\installer.msi
- follow instructions
- Get-Service Contrail* should return a stopped service with name "ContrailDockerDriver"
- .\installer.msi /x
- follow instructions
- Get-Service Contrail* should return nothing

File: wix.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "product": "Contrail docker driver",
+  "company": "Juniper",
```
Comment (mmasztalerczuk) [inline wix.json]: Are you sure that we should use that name here?

File: wix.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "license": "LICENSE.rtf",
+  "upgrade-code": "ca674931-876d-4dfb-bb67-136c585b6ffc",
+  "files": {
+    "guid": "ca8cbb44-2a0d-4f95-9375-dccabc92ffc1",
+    "items": [
+      "../../../../bin/contrail-windows-docker.exe"
```
Comment (mmasztalerczuk) [inline wix.json:9]: Could you use here some variable for "../../"? For example "[INSTALLDIR]\\contrail-windows-docker.exe\ " ?

**Review by mmasztalerczuk — APPROVED** [review]
It looks good. Only two questions.

File: wix.json (hunk 2 above)
Comment (m-kostrzewa) [reply wix.json:9]: This is a relative path given go workspace structure. I tried putting $Env:GOPATH or %GOPATH% here, but it doesn't work, unfortunately

File: wix.json (hunk 1 above)
Comment (m-kostrzewa) [reply wix.json]: I will change it to Juniper Networks

**Pull request merged**